misc/specs/atrium-specs-jvm/build.gradle

```diff
@@ -32,4 +32,5 @@ dependencies {
     //TODO #116 remove once there aren't any spek1 specs anymore
     api "org.jetbrains.spek:spek-api:$spek_version", excludeKotlin
     api mockito(), excludeKotlin
+    api niok(), excludeKotlin
```

Comment:
add the same to specs-android, should fix the build

Reply:
Done. It fixed the build.